services/121-service/src/main.ts

```diff
@@ -122,6 +122,8 @@ function generateSwaggerSummaryJson(app: INestApplication<any>): void {
 }
 
 async function bootstrap(): Promise<void> {
+  console.log(`Bootstrapping ${APP_TITLE} - ${APP_VERSION}`);
```

Comment:
Leave this log in here was a consious decision I suppose? @elwinschmitz

Reply:
Yeah. It wasn't necessary for this bug; But from the previous time we looked at the Docker-logs... I think its convenient to have the '_actual version_' somewhere readable..